js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "functional-test",
+  "version": "0.0.0",
+  "description": "A functional test for Permanent.org",
+  "scripts": {
+    "cypress:open": "cypress open",
+    "start": "cypress run",
+    "lint": "eslint .",
+    "test": "npm start"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PermanentOrg/functional-test.git"
+  },
+  "author": "Permanent.org",
+  "license": "AGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/PermanentOrg/functional-test/issues"
+  },
+  "homepage": "https://github.com/PermanentOrg/functional-test#readme",
+  "devDependencies": {
+    "eslint": "^8.46.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.1.0",
+    "eslint-config-prettier": "^8.9.0",
+    "eslint-plugin-cypress": "^2.13.3",
+    "eslint-plugin-import": "^2.28.0",
+    "eslint-plugin-prettier": "^5.0.0",
```

Comment:
This plugin means eslint will enforce all the prettier rules, right?

Reply:
Yes. :+1: